unit_tests/src/test_re_serialization.cpp

```diff
@@ -290,3 +290,16 @@ TEST_CASE("serialize_rsComm_ptr", "[pointer][serialization]")
         CHECK(out.at("option")          == rscp->option);
     }
 }
+
+TEST_CASE("serialize_dataObjInfo_ptr", "[pointer][serialization]")
+{
+    SECTION("#6123: resource id is serialized for REPF")
+    {
+        DataObjInfo doi{.rescId = 99999};
```

Comment:
investigate whether this zeroes out the memory of the other member variables?

Reply:
or write a full test that covers all of `DataObjInfo`.

Reply:
I am personally in favor of the second one although I realize this creates significantly more work :)

Reply:
agreed. the second option is definitely the right way to go.

Reply:
done.